manifests/satellite.pp

```diff
@@ -33,6 +10,7 @@
   $masquerade_domains    = undef,
   $masquerade_exceptions = undef,
 ) {
+  assert_private()
```

Comment:
This breaks CI.  A privte class with parameters is weird… probably not that private, or maybe a design flaw :smile:

Reply:
CI Tests reworked